src/data/user/index.ts

```diff
@@ -1 +1 @@
-export * from './userServices';
+export * from '../auth/userServices';
```

Comment:
Return userServices.ts file to the `data/user` folder. It shouldn't be in `auth` filder